api/v1beta1/provider_types.go

```diff
@@ -55,6 +55,7 @@ type ProviderStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=velaprovider
```

Comment:
```suggestion
// +kubebuilder:resource:shortName=vela-provider
```

Reply:
I didn't see any `-` ever appeared in a short name, did you?

Reply:
velapvd